### PR TITLE
Gauntlet: Aave V3 Polygon Freeze/Set LTV To 0

### PR DESCRIPTION
**Pull request by jbatch01**

### CODE REVIEW

File: src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Freeze and set LTV to 0 for DPI, BAL, CRV, and SUSHI on Aave v3 Polygon, 2024.01.19"
+author: "Gauntlet"
+discussions: "https://governance.aave.com/t/arfc-recommendation-to-freeze-and-set-ltv-to-0-on-low-cap-aave-v3-polygon-collateral-assets/16311"
+snapshot: https://snapshot.org/#/aave.eth/proposal/0x598f04c1f4fb2acba0a9bdaeb2b18e6e1f43b4c62845ee4b81ae5596a6fc2076
```
Comment (sakulstra) [inline src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119.md:5]: snapshot only supports a single value

File: src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119_20240130.sol [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * @author Gauntlet
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x8a190af80cffafcbca70727c807ef86933b2e08b5212b447eafab976a9612e75
+ * - Discussion: https://governance.aave.com/t/arfc-recommendation-to-freeze-and-set-ltv-to-0-on-low-cap-aave-v3-polygon-collateral-assets/16311
+ */
+contract AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119_20240130 is
+  AaveV3PayloadPolygon
+{
+  function _postExecute() internal override {
+    AaveV3Polygon.POOL_CONFIGURATOR.setReserveFreeze(AaveV3PolygonAssets.BAL_UNDERLYING, false);
+    AaveV3Polygon.POOL_CONFIGURATOR.setReserveFreeze(AaveV3PolygonAssets.CRV_UNDERLYING, false);
```
Comment (sakulstra) [inline src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119_20240130.sol:20]: this is the exact opposite of what you say on the aip
probably related to a bug in the generator: https://github.com/bgd-labs/aave-proposals-v3/blob/main/generator/features/freeze.ts#L35C17-L35C22 here it should not be `false` but `cfg.shouldBeFrozen` i guess. Perhaps you could patch it and include the patch here.

File: src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119.md (hunk 1 above)
Comment (jbatch01) [reply src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119.md:5]: Thanks - Do you have any preference for the snapshot link that we include?

File: src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119_20240130.sol (hunk 2 above)
Comment (jbatch01) [reply src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119_20240130.sol]: Good catch - I noticed this while generating the PR but it seemed to reflect what was from the generator. I pushed a patch for it in this PR.

**Review by kyzia551 — APPROVED** [review]

**Review by sakulstra — APPROVED** [review]

**Pull request merged**